wlauto/workloads/googlephotos/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -17,7 +18,7 @@
 
 public class UiAutomation extends UxPerfUiAutomation {
 
-    public static String TAG = "uxperf_googlephotos";
+    public static String TAG = "googlephotos";
```

Comment:
TAG is a bit of legacy left behind.  Typically it was used with logcat messages.  It was handy to prefix them all with uxperf so you only needed one filter in logcat to catch messages for all of our workloads.  I'd prefer we keep this as is for consistency.